src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -750,7 +750,7 @@ export default {
           filingId: header.filingId,
           filingAuthor: header.certifiedBy,
           filingDateTime: this.convertUTCTimeToLocalTime(header.date), // used for receipt
-          filingDate: this.convertUTCTimeToLocalTime(correction.correctedFilingDate),
+          filingDate: filingDate,
```

Comment:
You can just use the shorthand here (just put filingDate).

Reply:
While you're here, is `filingDateTime` also broken? And same solution?